domain/mvc/tokens.go

```diff
@@ -7,8 +7,23 @@ import (
 	"github.com/osmosis-labs/sqs/domain"
 )
 
+type TokensPoolLiquidityHandler interface {
+	// GetChainScalingFactorByDenomMut returns a chain scaling factor for a given denom
+	// and a boolean flag indicating whether the scaling factor was found or not.
+	// Note that the returned decimal is a shared resource and must not be mutated.
+	// A clone should be made for any mutative operation.
+	GetChainScalingFactorByDenomMut(denom string) (osmomath.Dec, error)
```

Comment:
Just questions:
1. What is scaling factor?
2. What does "Mut" stand for?

Reply:
Yes, if exponent in the [asset list](https://github.com/osmosis-labs/assetlists/blob/main/osmosis-1/generated/frontend/assetlist.json) is 6, the scaling factor is 10^6